src/pal/src/thread/process.cpp

```diff
@@ -2870,10 +2932,53 @@ void PROCNotifyProcessShutdown()
   Does not return
 --*/
 PAL_NORETURN
-void
+VOID
 PROCAbort()
 {
+    // Do any shutdown cleanup before aborting or creating a core dump
     PROCNotifyProcessShutdown();
+
+    // If enabled, launch the create minidump utility and wait until it completes
+    char* enabled = getenv("COMPlus_DbgEnableMiniDump");
```

Comment:
The getenv is not an async signal safe function and this code can be called from signal handlers. Can we get the setting from the env var e.g. during PAL initialization and store it to a global variable? The rest of the functions here look ok from this perspective.